Makefile.core.mk

```diff
@@ -405,12 +406,12 @@ gen-api-docs: ## Generate API documentation. Known issues: go fmt does not prope
 		--source-path=$(CRD_PATH) \
 		--templates-dir=$(TEMPLATES_DIR) \
 		--config=$(CONFIG_API_DOCS_GEN_PATH) \
-		--renderer=markdown \
+		--renderer=$(DOCS_RENDERER) \
 		--output-path=$(OUTPUT_DOCS_PATH) \
 		--output-mode=group
 	@find $(OUTPUT_DOCS_PATH) -type f -name "*.md" -exec sed -i 's/<br \/>/ /g' {} \;
 	@find $(OUTPUT_DOCS_PATH) -type f -name "*.md" -exec sed -i 's/\t/  /g' {} \;
```

Comment:
it looks like we will also need to replace the `\t`'s in the asciidoc, right?

Reply:
I just checked and you're right ! Fixing it